src/components/colorscale/color_attributes.js

```diff
@@ -64,14 +69,16 @@ module.exports = function makeColorScaleAttributes(context) {
         autocolorscale: extendDeep({}, colorScaleAttributes.autocolorscale, {
             description: [
                 'Has an effect only if `', context, '.color` is set to a numerical array.',
-                ' Determines whether or not the colorscale is picked using',
-                ' values inside `', context, '.color`.'
+                ' Determines whether the colorscale is the default palette (`autocolorscale: true`)',
+                ' or the palette determined by `', context, '.colorscale`.'
```

Comment:
This is a correction: in my experience, the default palette is being used if set to true (and whichever way it's set, the values inside the `color` array do result in different colors.)